File: Platron/Pages/Helpers/DemoshopHelper.cs
using System;
using Xunit;
using OpenQA.Selenium;
using OpenQA.Selenium.Support.UI;

namespace Platron.Pages.Helpers
{
    public class DemoshopHelper : BaseHelper
    {
        #region Buttons
        public AuthHelper ClickButtonEnter()
        {
            WhenIsClickable(UIDemoshopPage.ButtonEnter).Click();
            WaiteUntilPageLoad();
            driverHelper.SwitchTo().DefaultContent();
            driverHelper.SwitchTo().Frame(0);

            return AuthHelper;
        }

        public ShopCartHelper ClickInBinOnPage(int index = 1)
        {
            var elements = driverHelper.FindElements(By.XPath(UIDemoshopPage.ThingsOnMainPage));
            var inBinElement = WhenIsClickable(elements[index]).FindElement(By.XPath(UIDemoshopPage.InBinOnMainPage));
            inBinElement.Click();
            return ShopCartHelper;
        }
        #endregion
    }
}

File: Platron/src/Pages/DemoshopHelper.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;

import java.util.List;

public class DemoshopHelper extends  BaseHelper{
    public DemoshopHelper(){
        super();
    }
    public AuthHelper ClickButtonEnter()
    {
        WhenIsClickable(UIDemoshopPage.ButtonEnter).click();
        WaiteUntilPageLoad(10);
        driverHelper.switchTo().defaultContent();
        driverHelper.switchTo().frame(0);

        return new AuthHelper();
    }
    public ShopCartHelper ClickInBinOnPage(int index){
        List<WebElement> elements = driverHelper.findElements(By.xpath(UIDemoshopPage.ThingsOnMainPage));
        //System.out.println("elements " + elements.size());
        //DrawHighlight(elements.get(index));
        WebElement inBinElement = WhenIsClickable(elements.get(index)).findElement(By.xpath(UIDemoshopPage.InBinOnMainPage));
        //DrawHighlight(inBinElement);
        inBinElement.click();
        return new ShopCartHelper();
    }
}
